_topic_map.yml

```diff
@@ -165,7 +165,7 @@ Topics:
   File: generate-diagnostic-bundle
 - Name: Configuring endpoints
   File: configure-endpoints
-- Name: Monitoring with Prometheus
+- Name: Monitoring
```

Comment:
I am still seeing "Monitoring with Prometheus" in the left nav/TOC in the preview.

![Screenshot 2023-08-08 at 1 42 21 PM](https://github.com/openshift/openshift-docs/assets/96192119/142884c5-d4b6-4dba-b012-26099c4ca53e)

Reply:
Preview does not update for me :(